src/tools/clippy/tests/ui/attrs.stderr

```diff
@@ -37,13 +37,5 @@ LL | #![deny(clippy::restriction)]
    |
    = help: try enabling only the lints you really need
 
-error: restriction lints are not meant to be all enabled
```

Comment:
My guess is that the original inclusion here was intended to test that any level of blanket restriction enablement (including forbid) was covered, but given that the [code in question](https://github.com/rust-lang/rust-clippy/blob/master/clippy_lints/src/attrs.rs#L453) blanket covers everything except allow it feels like this should be sufficient (and adding a separate test does not feel worth it).